Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -18,8 +18,8 @@ namespace ACE.Command.Handlers
 {
     internal enum TestWeenieClassIds : uint
     {
-        Pants        = 120,
-        Tunic        = 134,
+        Pants = 120,
```

Comment:
I don't understand why you shifted these to non indented.  We've been keeping the assignments aligned on enums, I dislike the inconsistency this adds.

Reply:
*I* didn't change this purposefully, I think Visual Studio must've made this change for me when I did something. Weird. Fixed it, anyways.

Reply:
VS has been changing this automatically for me as well, if you use the CTRL+K and then CTRL+D shortcut, it will show you the same results...